src/components/inline-inputs/inline-inputs.spec.tsx

```diff
@@ -194,12 +194,31 @@ describe("Inline Inputs", () => {
   });
 
   describe("when a required prop is passed in", () => {
-    beforeEach(() => {
-      wrapper = render({ label: "foo", required: true });
+    it("sets required attribute for each input", () => {
+      const container = mount(
+        <InlineInputs required label="foo">
+          <input />
+          <input />
+          <input />
+        </InlineInputs>
+      );
+      const inputNodes = container.find("input") || [];
+
+      inputNodes.forEach((inlineInputField) => {
+        const inputNode = inlineInputField.getDOMNode();
+        expect(inputNode.hasAttribute("required")).toBe(true);
+      });
     });
 
-    it("then the isRequired prop should be passed to the label", () => {
-      expect(wrapper.find(Label).props().isRequired).toBe(true);
+    it("when there are no inputs set", () => {
+      const container = mount(
+        <InlineInputs required label="foo">
+          <div>foo</div>
+        </InlineInputs>
+      );
+
+      const label = container.find(Label);
+      expect(label.prop("isRequired")).toBe(true);
     });
```

Comment:
**comment**: I understand why we are trying to cover a sad path here, but from a user's perspective they won't be aware of React components and props, so I would remove this if its not needed for coverage.

```suggestion
```